src/manifest.json

```diff
@@ -56,7 +58,8 @@
         "https://*.ses.nsw.gov.au/Messages/Create",
         "https://*.ses.nsw.gov.au/Messages/Create/"
       ],
-      "js": ["contentscripts/messages/create.js"]
+      "js": ["contentscripts/messages/create.js"],
+      "run_at" : "document_end"
```

Comment:
Ill check, but I document_end runs before jquery.ready and is needed for the other JS in the page to pickup that the message.js code is there.